distributed/distributed.yaml

```diff
@@ -71,6 +71,15 @@ distributed:
     socket-backlog: 2048
     recent-messages-log-length: 0  # number of messages to keep for debugging
 
+    # Facebook's Zstandard - Fast real-time compression algorithm;  reference https://pypi.org/project/zstandard/
```

Comment:
This is probably not necessary

```suggestion
```

Reply:
it is a reminder comment for INTUITIVE DESIGN.
this comment let user know what is it,  let user make fast judgement by intuitive:
    whether they need to change the "zstd" config group.

keep it,  Harmless too.

Reply:
< The user can set this environment variable independently. I am not so sure that it is a good idea to do this for them.

we provide user a UNIFIED config system in ONE "dask.yaml" as UNIFIED as possible.
just like Direct Flight better than Transfer Flight to user;

Actually dask|distributed is an accurate complex system with so many configs;
UNIFIED and INTUITIVE DESIGN is very useful.

Reply:
> we provide user a UNIFIED config system in ONE "dask.yaml" as UNIFIED as possible.

Unfortunately, Dask connects with many other software libraries.  We can not be responsible for all configuration systems.  We need to keep scope small for maintenance reasons.  If another library already has a configuration solution then we point users to that system.  Otherwise our configuration system becomes very very large.

Reply:
unified system makes user easy. 
scope small make maintenance easy;
I choose the First one. 

large configuration system is not reason. dask with distributed is large already.

we do need to rewrite the rules.
for user,  dask team.

Reply:
Please stop marking these as resolved.  Please remove these comments.

Reply:
@mrocklin
Please keep these comments. 
Just like you keep the COPYRIGHT document from PSF,  in GPL, Apache, BSD, MIT;

we are NOT ONLY OPEN SOURCE;
BUT ALSO OPEN CULTURE AND BUILD BETTER GLOBAL COMMUNITY. 

To achieve that,  you do need to 
EMBRACE these GLOBAL OPEN CULTURE and CONCEPTS of VALUES are not identical to your's.